app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -667,6 +722,10 @@ private void dispose(@Nullable Disposable disposable) {
                 roomsQueryDisposable != null && !roomsQueryDisposable.isDisposed()) {
             roomsQueryDisposable.dispose();
             roomsQueryDisposable = null;
+        } else if (disposable == null &&
+            openConversationsQueryDisposable != null && !openConversationsQueryDisposable.isDisposed()) {
+            openConversationsQueryDisposable.dispose();
+            openConversationsQueryDisposable = null;
         }
```

Comment:
i copied the roomsQueryDisposable stuff and named it openConversationsQueryDisposable for the open conversations.
but tbh i have no idea yet why there is this differentiation between the parameter "disposable"(line 717) and roomsQueryDisposable.

Reply:
Seems to be a lazy function handling two things: disposing disposable and if this is null, it tries to dispose the roomsQueryDisposable property, which is only assigned in L464.
TBH, I currently do not see why we need this roomsQueryDisposable at all…
It seems to be just needed due to framework reasons…

Reply:
> TBH, I currently do not see why we need this roomsQueryDisposable at all…

that's the point. I could refactor it be similar to most other uses of rxjava in talk-android. But there are some different variants used. Might make sense / might be random / might have to learn more about rxjava. Would like to talk about RxJava in the next android meeting.. So at least until then i'll keep it like it is.